app/models/role.rb

```diff
@@ -49,17 +57,17 @@ def reviewer?
     name == 'reviewer'
   end
 
-  private
-
-  def permision?
-    !permission.nil?
-  end
-
   def predefined_user?
     [admin?, redactor?, reviewer?].include?(name)
   end
 
   def editor?
     [admin?, redactor?].include?(name)
   end
+
+  private
+
+  def permision?
+    !permission.nil?
+  end
```

Comment:
It s very RUDE. Please use git more better

Reply:
It means that predefined_user and editor methods become not private